week3/homework/Controllers/ConverterController.cs

```diff
@@ -0,0 +1,81 @@
+using Microsoft.AspNetCore.Mvc;
+using homework.Models;
+
+namespace homework.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ConverterController : ControllerBase
+{
+
+
+    [HttpGet("GallonsToLiters")]
+    public double GallonsToLiters(int gallons)
+    {
+        return gallons * 3.785;
+    }
+
+    [HttpGet("ConvertMiles")]
+    public Distance ConvertMiles(int miles)
+    {
+        double km = miles * 1.609;
+        return new Distance() { Miles = miles, Kilometers = km };
+    }
+
+    [HttpPost("ConvertValues")]
+    public Object[] ConvertValues([FromBody] ConversionRequest Request)
```

Comment:
Will work, but generally you should avoid returning Object and aim for something more strongly typed. The suggested solution here would be to return a list of value-objects which each had a type and a double value.

Reply:
Yes, that what I wanted to do, but I did not know how to do it :(